admin/tabs/general/GeneralTab.js

```diff
@@ -21,9 +21,14 @@ export const generalTab = hoistCmp.factory(
             tabs: [
                 {id: 'about', icon: Icon.info(), content: aboutPanel},
                 {id: 'config', icon: Icon.settings(), content: configPanel},
-                {id: 'users', icon: Icon.users(), content: userPanel},
+                {id: 'users', icon: Icon.users(), content: userPanel, omit: omitTab('hideUsersTab')},
                 {id: 'alertBanner', icon: Icon.bullhorn(), content: alertBannerPanel}
             ]
         }
     })
 );
+
+const omitTab = (field) => {
+    const conf = XH.getConf('xhAdminConsoleConfig', {});
```

Comment:
Can we call it `xhAdminAppConfig`?  Can we add a default value in hoist core, so easy for new app users to find it?   

Also this function should just be called `hideUsersTab()` doesn't seem like it is ever going to be a general tab hiding mechanism